MyStore/src/JsonParcer.cpp

```diff
@@ -0,0 +1,77 @@
+#include "JsonParcer.h"
+
+
+bool JsonParcer::Deserialization()
+{
+	std::ifstream file(this->m_file_path);

```

Comment:
this-> is unnecessary here, you can access m_file_path directly. The same in other places in this function.